test/test_embedded/main.cpp

```diff
@@ -0,0 +1,24 @@
+#include <unity.h>
+
+#include "test_sidereal.h"
+
+// void setup()
+// {
+
+// }
+
+// void loop()
+// {
+
+// }
+
+int main(int argc, char **argv)
+{
+    UNITY_BEGIN();
+
+    test::sidereal::run();
```

Comment:
Just curious, are these tests actually running on the target? @andre-stefanov had some trouble running embedded tests

Reply:
Reminder to remove these files too